src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$time-seconds: 9s;
+$time-minute: 540s;
```

Comment:
why this numbers
there is 60s in minute

Reply:
60 seconds is long to wait, I just made arrows faster